source/help/messaging/formatting-text.rst

```diff
@@ -235,3 +235,19 @@ Create a table by placing a dashed line under the header row and separating the
 Renders as:
 
 .. image:: ../../images/markdownTable1.PNG
+
+
+Math Formulas
```

Comment:
I disagree. The fact that it is done using a `Code Block` is just an implementation detail. None of the features associated with a `Code Block` are available in math mode: syntax highlighting, themes, or the 4-space indented codeblock syntax.

Also, think of it like this: A user tries to search the docs for how to typeset a formula. The fact that it is done using a `Code Block` is most likely not known to the user. So putting the feature under `Code Block` actually obscures the information by putting it under a seemingly unrelated headline.

The way it is formulated now improves discoverability while highlighting the detail that is technically a `Code Block`.